[INFO -- 2018-11-25 21:08:13]
Starting flip_value script
version: 1.1
[INFO -- 2018-11-25 21:08:13]
Initialization strings:
[INFO -- 2018-11-25 21:08:13]
(1) file /tmp/lreg/lreg
[INFO -- 2018-11-25 21:08:13]
(2) set args /tmp/lreg/randInput /tmp/lreg/output /tmp/lreg/detected.log
[INFO -- 2018-11-25 21:08:14]
initSignal:0
[INFO -- 2018-11-25 21:08:14]
endSignal:6
[INFO -- 2018-11-25 21:08:14]
faultModel:1
[EXCEPTION -- 2018-11-25 21:08:15]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[EXCEPTION -- 2018-11-25 21:08:15]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-11-25 21:08:18]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec6801 in __GI_abort () at abort.c:79
#2  0x00007ffff6f0f897 in __libc_message (action=action@entry=do_abort, fmt=fmt@entry=0x7ffff703cb9a "%s\n") at ../sysdeps/posix/libc_fatal.c:181
#3  0x00007ffff6f1690a in malloc_printerr (str=str@entry=0x7ffff703e7a8 "munmap_chunk(): invalid pointer") at malloc.c:5350
#4  0x00007ffff6f1decc in munmap_chunk (p=0x5555557725c0) at malloc.c:2846
#5  __GI___libc_free (mem=0x5555557725d0) at malloc.c:3117
#6  0x0000555555557dac in __gnu_cxx::new_allocator<double>::deallocate (this=0x7fffffffd920, __p=0x5555557725d0) at /usr/include/c++/7/ext/new_allocator.h:125
#7  0x00005555555575bb in std::allocator_traits<std::allocator<double> >::deallocate (__a=..., __p=0x5555557725d0, __n=16) at /usr/include/c++/7/bits/alloc_traits.h:462
#8  0x0000555555556c66 in std::_Vector_base<double, std::allocator<double> >::_M_deallocate (this=0x7fffffffd920, __p=0x5555557725d0, __n=16) at /usr/include/c++/7/bits/stl_vector.h:180
#9  0x0000555555557b5d in std::vector<double, std::allocator<double> >::_M_realloc_insert<double> (this=0x7fffffffd920, __position=0, __args#0=@0x7fffffffd698: 19328.177037179787) at /usr/include/c++/7/bits/vector.tcc:448
#10 0x0000555555557264 in std::vector<double, std::allocator<double> >::emplace_back<double> (this=0x7fffffffd920, __args#0=@0x7fffffffd698: 19328.177037179787) at /usr/include/c++/7/bits/vector.tcc:105
#11 0x0000555555556880 in std::vector<double, std::allocator<double> >::push_back (this=0x7fffffffd920, __x=@0x7fffffffd698: 19328.177037179787) at /usr/include/c++/7/bits/stl_vector.h:954
#12 0x0000555555555a8d in execute (input=0x7fffffffe1f1 "/tmp/lreg/randInput") at lreg.cpp:160
#13 0x0000555555555cb0 in main (argc=4, argv=0x7fffffffde48) at lreg.cpp:196

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Fliping value of field: _M_current
Fliping a bit of the pointer
Memory content before bitflip:0x500x260x770x550x550x550x000x00
Fault Model: Double bit-flip
base address to flip value: 0x7fffffffd590
address max offset: 8
Memory content after  bitflip:0x500x260x770x550x5c0x550x000x00
frame name: std::vector<double, std::allocator<double> >::_M_realloc_insert<double>
symbol name: __position
symbol filename: /usr/include/c++/7/bits/vector.tcc
symbol line: 395
value: <error reading variable: Cannot access memory at address 0x555c55772650>
value address: 0x7fffffffd590
Type: The type is a structure.
Type sizeof: 8
Field name: _M_current
Field Type: The type is a pointer.
Field Type sizeof: 8

[INFO -- 2018-11-25 21:08:18]
Fault Injection Successful after 4.416085720062256s
[INFO -- 2018-11-25 21:08:18]
event type: exit
[EXCEPTION -- 2018-11-25 21:08:18]
exit code: no exit code available
